bento/src/main/java/com/yelp/android/bento/componentcontrollers/TabViewPagerComponentViewHolder.kt

```diff
@@ -0,0 +1,82 @@
+package com.yelp.android.bento.componentcontrollers
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout.LayoutParams
+import androidx.viewpager.widget.ViewPager
+import androidx.viewpager.widget.ViewPager.OnPageChangeListener
+import com.google.android.material.tabs.TabLayout
+import com.yelp.android.bento.R
+import com.yelp.android.bento.componentcontrollers.TabViewPagerComponentViewHolder.TabViewPagerComponentViewHolderData
+import com.yelp.android.bento.components.TabViewPagerComponent.TabViewPagerComponentViewModel
+import com.yelp.android.bento.core.ComponentViewHolder
+
+class TabViewPagerComponentViewHolder : ComponentViewHolder<OnPageChangeListener, TabViewPagerComponentViewHolderData>() {
+
+    private lateinit var mContext: Context
+    private lateinit var mContainer: View
+    private lateinit var tabLayout: TabLayout
+    private lateinit var viewPager: ViewPager
+
+    override fun inflate(parent: ViewGroup): View {
+        mContext = parent.context
+        val view = LayoutInflater.from(parent.context)
+                .inflate(R.layout.bento_component_tab_view_pager, parent, false)
+        mContainer = view.findViewById(R.id.tab_view_pager_parent)
+        tabLayout = view.findViewById(R.id.tab_layout)
+        viewPager = view.findViewById(R.id.view_pager)
+
+        tabLayout.setupWithViewPager(viewPager)
+        viewPager.offscreenPageLimit = Integer.MAX_VALUE
```

Comment:
Do we really want MAX_VALUE here? This is going to cause the view page to hold _all_ pages in memory. If the pager has tons of pages this could easily lead to an OOM.